packages/core/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@x-filter/core",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "dependencies": {
+    "typescript": "latest"
```

Comment:
why not have `pnpm.lock` & `.gitignore` files

Reply:
`.gitignore` is in the root directory.

Reply:
you need install it, avoid potential conflict

Reply:
updated

Reply:
and .gitignore file

Reply:
Not required, cause it in the root directory.